scripts/default/integtest.sh

```diff
@@ -89,7 +89,7 @@ fi
 
 if [ -z "$CREDENTIAL" ]
 then
-  CREDENTIAL="admin:admin"
+  CREDENTIAL="admin:myStrongPassword123!"
```

Comment:
This wouldn't work for OS version lower than 2.12. 
It may not make much sense to run integration tests for released 2.x versions but it will definitely break integ tests for 1.x line.

Reply:
Right! The code is generic for all versions as of today. Until we solve the branching problem.

Reply:
To unblock this we can try adding if/else condition to differentiate with 1.x and above until we have this branching/versioning for the build repo solved. The script already has an argument `OPENSEARCH_VERSION` so adding if/else should be straight forward.

Reply:
I think issue with that is what if patch version 2.0 < patch version < 2.12 come along? I was looking into bash script to compare semantic version, but it seems complex, not sure how do you folks feel about adding something like this: https://stackoverflow.com/questions/4023830/how-to-compare-two-strings-in-dot-separated-version-format-in-bash?

Reply:
@rishabh6788 @gaiksaya @prudhvigodithi I changed the logic in the script itself to set a strong password if the version is after 2.12.0. Let me know if that makes sense to keep. I also included this logic in the integtest.sh files of repo's which have their own versions of integtest.sh.